### PR TITLE
Adds an info log when test-all-containers is false

### LINKED ISSUE #31: Give info statement on runtime if test-all-containers is false
If the argument `test-all-containers` is false (default) , chances are that it won't process a thing. Therefore place a clear info message in the logs that it's running on false. That gives a better indication on what is going on.

### PR DESCRIPTION
**Pull request by wiardvanrij**
fixes #31

Signed-off-by: Wiard van Rij <w.vanrij@fullstaq.com>

### CODE REVIEW

File: cmd/app/app.go [hunk 1]
```diff
@@ -87,6 +87,10 @@ func NewCommand(ctx context.Context) *cobra.Command {
 				}
 			}()
 
+			if !opts.DefaultTestAll {
+				log.Info("Argument test-all-containers is FALSE. Only containers with the annotation 'enable.version-checker.io' will be parsed")
```
Comment (JoshVanL) [inline cmd/app/app.go:91]: nit: "flag --test-all-containers=%t so only containers with the annotation 'enable.version-checker.io' will be parsed"

**Review by JoshVanL — CHANGES_REQUESTED** [review]
Thanks for adding this, I have been caught out a few times myself :sweat_smile:

File: cmd/app/app.go (hunk 1 above)
Comment (wiardvanrij) [reply cmd/app/app.go]: Nice. I've even enriched it a bit more. Let me know if that's too much..

Comment (JoshVanL) [reply cmd/app/app.go]: No that's great! 

Can we also have the same `only containers with the annotation "enable.version-checker.io/${my-container}=true" will be parsed` for when the flag is false.

Also remove the brackets around `opts.DefaultTestAll`

**Comment by JoshVanL** [comment]
Cheers @wiardvanrij !

/lgtm

**Pull request merged**